### PR TITLE
 anatoli-zinhan - tries update master

### PR DESCRIPTION
**Pull request by zingan-anatolii**

### CODE REVIEW

**Review by zan-dev-res — COMMENTED** [review]
I am fine with it

File: hello.js [hunk 1]
```diff
@@ -1 +1 @@
-console.log('123-merged-12')
\ No newline at end of file
+console.log('123-merged-12-')
```
Comment (zan-dev-res) [inline hello.js]: it is ok

**Comment by zingan-anatolii** [comment]
oh

**Comment by zingan-anatolii** [comment]
> I am fine with it

ok

File: hello.js (hunk 1 above)
Comment (zan-dev-res) [reply hello.js]: 1

Comment (zan-dev-res) [inline hello.js]: add plus symbol, please

Comment (zingan-anatolii) [reply hello.js]: done

**Review by zan-dev-res — APPROVED** [review]
fine

**Pull request merged**